src/maptm/src/maptm_config.c

```diff
@@ -450,8 +481,14 @@ bool config_mapt(void)
         return false;
     }
 
-    /* Hard Coded value: Must be changed */
-    char subnetcidr4[20] = "192.168.1.1/24";
+    // Get LAN Subnet
+    char subnetcidr4[20] = "";
+    set_subnetcidr4(subnetcidr4);
+    if (subnetcidr4 == NULL)
+    {
+        LOGE("Unable to configure MAP-T option Unable to get subnetcidr4");
```

Comment:
```suggestion
        LOGE("Unable to get subnetcidr4");
```

or

```suggestion
        LOGE("Unable to configure MAP-T option (unable to get subnetcidr4)");
```

whichever you think is more appropriate